### PR TITLE
fix: flappy Slider/Color Area tests

### PR DESCRIPTION
**Pull request by hunterloftis**
## Description

- replace `elementUpdate` with `nextFrame`, which seems more reliable, in intermittently failing tests
- fix visibility of `ariaValueText` in Slider to be compatible with its type signature

## Related issue(s)

- https://github.com/adobe/spectrum-web-components/issues/1772

## Motivation and context

While developing https://github.com/adobe/spectrum-web-components/pull/1771, I noticed that these tests would intermittently fail. The failures were rare and non-deterministic, which makes it difficult to know whether or not your work created a regression.

## How has this been tested?

Experimentation (by running `yarn test` repeatedly) suggests that this PR will more-reliably `await` the state changes that the tests are confirming.

## Types of changes

-   [X] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [ X] My code follows the code style of this project.
-   [ X] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [ X] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ X] I have added tests to cover my changes.
-   [ X] All new and existing tests passed.

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

**Comment by Westbrook** [comment]
I'm a little uncomfortable with needing to move to `nextFrame` here, however due to the fact that we're leveraging the `executeCommand` API to communicate back and forth between the client side tests and the server side browser runner, I could be convinced that this is OK.

Context: `await elementUpdated(el);` is an abstraction on `await el.updateComplete;` which is what a consuming developer would normally wait for in this context. If that timing isn't correct, we should confirm whether there is additional work inside of the element to ensure that this workflow is stable for users that need to leverage it in their code. An alternative could be to leverage the `oneEvent` helper, to wait for expected events in the user workflow before we test the current realities fo the code. This may also help us to fully confirm that the validation is happening synchronous to the events dispatched as expected.

**Comment by hunterloftis** [comment]
I think that's a reasonable concern. The reason I suspect `nextFrame` may be ok is that clues indicate that the failures are due to our testing framework rather than the underlying library. _Most_ of the time, the tests pass with `elementUpdated`, suggesting a race condition in transferring updated state between the browser and the testing infrastructure.

I'll change this to check state after the events are received instead; it'd be good to confirm that the updated state is available synchronously.

**Comment by hunterloftis** [comment]
@Westbrook this is now both event-based and rebased!

File: packages/slider/test/slider.test.ts [hunk 1]
```diff
@@ -14,12 +14,30 @@ import '../sp-slider.js';
...
+    nextFrame,
+} from '@open-wc/testing';
 import { sendKeys, executeServerCommand } from '@web/test-runner-commands';
-import { spy } from 'sinon';
 import { ProvideLang } from '@spectrum-web-components/theme';
 
 describe('Slider', () => {
+    // At least one browser (Webkit) maintains shared global state for the mouse,
+    // such that `send-mouse` type: 'down' without subsequently sending 'up'
+    // can prevent a subsequent test from sending 'down' correctly.
```
Comment (Westbrook) [inline packages/slider/test/slider.test.ts:31]: 👏🏼 👏🏼 👏🏼 👏🏼

**Review by Westbrook — APPROVED** [review]
lgtm

**Pull request merged**